frontend/src/app/components/add-ipo/add-ipo.component.ts

```diff
@@ -0,0 +1,144 @@
+import { Component, OnInit } from '@angular/core';
+import { Company } from 'src/app/models/company-model';
+import { Exchange } from 'src/app/models/exchange-model';
+import { Ipo } from 'src/app/models/ipo-model';
+import { AuthService } from 'src/app/services/auth.service';
+import { CompanyService } from 'src/app/services/company.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
import { CompanyService } from "src/app/services/company.service";
```